minio/datatypes.py

```diff
@@ -230,33 +235,51 @@ def fromxml(cls, element, bucket_name, is_delete_marker=False):
         )
 
 
-def parse_list_objects(response, bucket_name):
+def parse_list_objects(response, bucket_name=None):
     """Parse ListObjects/ListObjectsV2/ListObjectVersions response."""
     element = ET.fromstring(response.data.decode())
+    bucket_name = findtext(element, "Name", True)
```

Comment:
This is what I was talking about.

If we set the `bucket_name` here no matter what, why do we have it as an input parameter?